addons/event/views/event_views.xml

```diff
@@ -469,5 +479,23 @@
 
         <menuitem name="Mail Schedulers" id="menu_event_type" action="action_event_mail" parent="menu_event_configuration" groups="base.group_no_one"/>
 
+        <record id="action_server_notify_schedule_updatation" model="ir.actions.server">
+            <field name="name">Send mail</field>
+            <field name="condition">True</field>
+            <field name="type">ir.actions.server</field>
+            <field name="model_id" ref="model_event_registration" />
+            <field name="state">code</field>
+            <field name="code">self.notify_schedule_updation(cr, uid, context.get('active_ids', []))</field>
```

Comment:
no need to check for active_mode and active_ids? 
like : 
 if context.get('active_model') == 'event.registration' and context.get('active_ids'):
       object.notify_schedule_updation(context['active_ids'])